packages/vuetify/src/components/VTimePicker/VTimePickerClock.ts

```diff
@@ -197,26 +197,36 @@ export default mixins<options &
       const coords = { x: clientX - left, y: top - clientY }
       const handAngle = Math.round(this.angle(center, coords) - this.rotate + 360) % 360
       const insideClick = this.double && this.euclidean(center, coords) < (innerWidth + innerWidth * this.innerRadiusScale) / 4
+
+      for (let i = 0; i < Math.ceil(15 / this.degreesPerUnit); i++) {
+        let value
+
+        value = this.angleToValue(handAngle + i * this.degreesPerUnit, insideClick)
+        if (this.isAllowed(value)) return this.setMouseDownValue(value)
+
+        value = this.angleToValue(handAngle - i * this.degreesPerUnit, insideClick)
+        if (this.isAllowed(value)) return this.setMouseDownValue(value)
+      }
+    },
+    angleToValue (angle: number, insideClick: boolean): number {
       const value = (
-        Math.round(handAngle / this.degreesPerUnit) +
+        Math.round(angle / this.degreesPerUnit) +
         (insideClick ? this.roundCount : 0)
       ) % this.count + this.min
 
       // Necessary to fix edge case when selecting left part of the value(s) at 12 o'clock
-      let newValue: number
-      if (handAngle >= (360 - this.degreesPerUnit / 2)) {
-        newValue = insideClick ? this.max - this.roundCount + 1 : this.min
+      if (angle >= (360 - this.degreesPerUnit / 2)) {
+        return insideClick ? this.max - this.roundCount + 1 : this.min
       } else {
-        newValue = value
+        return value
       }
-
-      if (this.isAllowed(value)) {
-        if (this.valueOnMouseDown === null) {
-          this.valueOnMouseDown = newValue
-        }
-        this.valueOnMouseUp = newValue
-        this.update(newValue)
+    },
+    setMouseDownValue (value: number) {
+      if (this.valueOnMouseDown === null) {
+        this.valueOnMouseDown = value
       }
+      this.valueOnMouseUp = value
```

Comment:
As outlined in the guide above, should have a blank line after the close of the conditional. While these lines are not returning, I think we should apply this here as well. If you agree, I'll update the Coding Guidelines to reflect this standard.

Reply:
> I'll update the Coding Guidelines to reflect this standard
 
👍

Reply:
updated